packages/truffle-core/lib/commands/develop.js

```diff
@@ -14,102 +15,83 @@ var command = {
     usage: "truffle develop",
     options: []
   },
-  runConsole: function(config, ganacheOptions, done) {
-    var Console = require("../console");
-    var Environment = require("../environment");
+  async runConsole(config, ganacheOptions, done) {
+    const Console = require("../console");
+    const Environment = require("../environment");
```

Comment:
Non-standard variable naming with leading capital. See above, feel free to ignore/resolve.